src/dotnet/Chat.UI.Blazor/Components/ChatListItemIcon.razor

```diff
@@ -0,0 +1,36 @@
+@switch (Chat.ChatType) {
+    case ChatType.Group:
+        <Pic Title="@Chat.Title" ContentId="@Chat.Picture"/>
+        break;
+    case ChatType.Peer:
+        <UserAuthorCircle ShowsPresence="true" UserId="@_peerChatTargetUserId"/>
+        break;
+    default:
+        throw new ArgumentOutOfRangeException(nameof(Chat.ChatType), Chat.ChatType, null);
+}
+
+@code {
+    private Symbol _peerChatTargetUserId;
+    [Inject] private IAccounts Accounts { get; init; } = null!;
+    [Inject] private Session Session { get; init; } = null!;
+
+    [Parameter, EditorRequired, ParameterComparer(typeof(ByReferenceParameterComparer))]
+    public Chat Chat { get; init; } = null!;
+
+    protected override async Task OnParametersSetAsync() {
+        _peerChatTargetUserId = await GetPeerChatTargetUserId().ConfigureAwait(false);
```

Comment:
This is clearly a mistake.

Reply:
You can't access or assign component fields after .ConfigureAwait(false).

Reply:
done